rest/route/project.go

```diff
@@ -1011,6 +1014,87 @@ func (h *getProjectTasksHandler) Run(ctx context.Context) gimlet.Responder {
 	return gimlet.NewJSONResponse(versions)
 }
 
+////////////////////////////////////////////////////////////////////////
+//
+// GET /rest/v2/projects/{project_id}/tasks/executions
+
+type getProjectTaskExecutionsHandler struct {
+	projectName string
+	opts        model.GetProjectTaskExecutionReq
+
+	startTime time.Time
+	endTime   time.Time
+	projectId string
+}
+
+func makeGetProjectTaskExecutionsHandler() gimlet.RouteHandler {
+	return &getProjectTaskExecutionsHandler{}
+}
+
+func (h *getProjectTaskExecutionsHandler) Factory() gimlet.RouteHandler {
+	return &getProjectTaskExecutionsHandler{}
+}
+
+func (h *getProjectTaskExecutionsHandler) Parse(ctx context.Context, r *http.Request) error {
+	body := utility.NewRequestReader(r)
+	defer body.Close()
+
+	var err error
+	if err = utility.ReadJSON(r.Body, &h.opts); err != nil {
+		return errors.Wrap(err, "reading from JSON request body")
+	}
+
+	if h.opts.BuildVariant == "" || h.opts.TaskName == "" {
+		return errors.New("'build_variant' and 'task_name' are required")
+	}
+	h.startTime, err = model.ParseTime(h.opts.StartTime)
+	if err != nil {
+		return errors.Wrapf(err, "parsing 'start_time' %s", h.opts.StartTime)
+	}
+
+	// If not specified, get the latest information.
```

Comment:
I was initially confused by the verbiage of the comment because it says `If not specified` but it's above the case where it is specified, (!=""). I wonder if there's a better way to phrase it (although I can't think of one, lol).

Reply:
thank you!